cli/tauri-cli/src/bundle/wix.rs

```diff
@@ -60,6 +60,52 @@ struct ExternalBinary {
   path: String,
 }
 
+#[derive(Serialize)]
+struct ResourceDirectory {
+  name: String,
+  files: Vec<ResourceFile>,
+  directories: Vec<ResourceDirectory>
+}
+
+impl ResourceDirectory {
+  fn add_file(&mut self, file: ResourceFile) {
+    self.files.push(file);
+  }
+
+  // generates the wix XML string to bundle this directory resources recursively
+  fn get_wix_data(self) -> (String, Vec<String>) {
```

Comment:
This here should return a `crate::Result<(String, Vec<String>)>`.  Gives more options for trouble shooting and error handling.